### ISSUE TITLE
Having trouble getting config.txt to work

### ISSUE THREAD

**Issue by IronCraftMan** [MultiPoolMiner/MultiPoolMiner#2347]
So I'm trying to set up mining on MPH, NiceHash, and Zpool. I'm using Start.bat and have removed all arguments from that, and I've attempted to set up my Config.txt. However, it doesn't seem to be mining to any of my accounts. The web GUI doesn't list any of my MPH balance stats, and MPH and NiceHash don't list my worker as working.

 This is a (partially redacted) copy of my Config.txt. I'm not sure what I'm doing wrong.
```
{
    "Algorithm": "$Algorithm",
    "AllowedBadShareRatio": "$AllowedBadShareRatio",
    "API_ID": "$API_ID",
    "API_Key": "<-->",
    "APIPort": "$APIPort",
    "BasePowerUsage": "50",
    "BenchmarkInterval": "$BenchmarkInterval",
    "CoinName": "$CoinName",
    "ConfigFile": "$ConfigFile",
    "CreateMinerInstancePerDeviceModel": true,
    "Currency": "usd",
    "Dashboard": "$Dashboard",
    "Debug": "$Debug",
    "Delay": "$Delay",
    "DeviceName": "nvidia",
    "DisableDevFeeMining": "$DisableDevFeeMining",
    "DisableMinersWithDevFee": "$DisableMinersWithDevFee",
    "ErrorAction": "$ErrorAction",
    "ErrorVariable": "$ErrorVariable",
    "ExcludeAlgorithm": "$ExcludeAlgorithm",
    "ExcludeCoinName": "$ExcludeCoinName",
    "ExcludeDeviceName": "$ExcludeDeviceName",
    "ExcludeMinerName": "$ExcludeMinerName",
    "ExcludeMiningCurrency": "$ExcludeMiningCurrency",
    "ExcludePoolName": "miningpoolhubcoins",
    "HashRateSamplesPerInterval": "$HashRateSamplesPerInterval",
    "HWiNFO64_SensorMapping": "$HWiNFO64_SensorMapping",
    "IgnoreFees": "$IgnoreFees",
    "IgnorePowerCost": "$IgnorePowerCost",
    "InformationAction": "$InformationAction",
    "InformationVariable": "$InformationVariable",
    "Interval": "$Interval",
    "MeasurePowerUsage": "$MeasurePowerUsage",
    "MinerName": "$MinerName",
    "MinerStatusKey": "$MinerStatusKey",
    "MinerStatusUrl": "$MinerStatusUrl",
    "MinHashRateSamples": "$MinHashRateSamples",
    "MiningCurrency": "$MiningCurrency",
    "MinWorker": "$MinWorker",
    "OutBuffer": "$OutBuffer",
    "OutVariable": "$OutVariable",
    "PipelineVariable": "$PipelineVariable",
    "PoolBalancesUpdateInterval": "$PoolBalancesUpdateInterval",
    "PoolName": "$PoolName",
    "PowerPrices": "$PowerPrices",
    "PricePenaltyFactor": "$PricePenaltyFactor",
    "ProfitabilityThreshold": "$ProfitabilityThreshold",
    "Proxy": "$Proxy",
    "Region": "US",
    "ReportStatusInterval": "$ReportStatusInterval",
    "ShowAllMiners": "$ShowAllMiners",
    "ShowAllPoolBalances": "$ShowAllPoolBalances",
    "ShowMinerWindow": "$ShowMinerWindow",
    "ShowPowerUsage": "$ShowPowerUsage",
    "SingleAlgoMining": "$SingleAlgoMining",
    "SSL": "$SSL",
    "SwitchingPrevention": "$SwitchingPrevention",
    "UserName": "<-MiningPoolHubName->",
    "Verbose": "$Verbose",
    "Wallet": "<-MyBitcoinWallet->",
    "WarmupTime": "$WarmupTime",
    "WarningAction": "$WarningAction",
    "WarningVariable": "$WarningVariable",
    "Watchdog": "$Watchdog",
    "WorkerName": "<-->",
    "Pools": {
        "MiningPoolHub": {
            "Region": "US",
            "API_Key": "<-->",
            "UserName": "<-MPHName->",
			"WorkerName": "<-->"
        },
        "NiceHash": {
            "Wallets": {
                "BTC": "<-NiceHashBTCWallet->"
            },
            "IsInternalWallet": true,
			"Region": "US",
			"WorkerName": "<-->"
        }
    },
    "Wallets": {
        "BTC": "<-MyBTCWallet->"
    },

    "VersionCompatibility": "3.3.0"
}

```

**Comment by UselessGuru** [comment]
The file looks OK.

What are the parameters you use to start MPM?
Please post the output of 'http://localhost:3999/runningconfig.html' when MPM is running.

**Comment by IronCraftMan** [comment]
> The file looks OK.
> 
> What are the parameters you use to start MPM?
> Please post the output of 'http://localhost:3999/runningconfig.html' when MPM is running.

Here it is:
```
{
    "Algorithm": [
        "Blake2s",
        "Cuckaroo39",
        "Cuckatoo31",
        "CryptonightR",
        "CryptonightV8",
        "CryptonightHeavy",
        "Decrednicehash",
        "Ethash",
        "Ethash2GB",
        "Ethash3GB",
        "Equihash",
        "Equihash1445",
        "Equihash1505",
        "Keccak",
        "Lbry",
        "Lyra2re3",
        "Mtp",
        "MtpNiceHash",
        "Neoscrypt",
        "Pascal",
        "Sib",
        "Skein",
        "Skunk",
        "Sonoa",
        "Timetravel10",
        "X16R",
        "X16Rt",
        "X16s",
        "X17",
        "X22i"
    ],
    "AllowedBadShareRatio": 0,
    "API_ID": 0,
    "API_Key": "<-MPHAPIKey->",
    "APIPort": 3999,
    "BasePowerUsage": "50",
    "BenchmarkInterval": 60,
    "CoinName": null,
    "ConfigFile": ".\\Config.txt",
    "CreateMinerInstancePerDeviceModel": true,
    "Currency": "usd",
    "Dashboard": true,
    "Debug": null,
    "Delay": 0,
    "DeviceName": "nvidia",
    "DisableDevFeeMining": false,
    "DisableMinersWithDevFee": false,
    "ErrorAction": null,
    "ErrorVariable": null,
    "ExcludeAlgorithm": null,
    "ExcludeCoinName": null,
    "ExcludeDeviceName": null,
    "ExcludeMinerName": null,
    "ExcludeMiningCurrency": null,
    "ExcludePoolName": "miningpoolhubcoins",
    "HashRateSamplesPerInterval": 10,
    "HWiNFO64_SensorMapping": null,
    "IgnoreFees": false,
    "IgnorePowerCost": false,
    "InformationAction": null,
    "InformationVariable": null,
    "Interval": 60,
    "MeasurePowerUsage": false,
    "MinerName": null,
    "MinerStatusUrl": "https://multipoolminer.io/monitor/miner.php",
    "MinHashRateSamples": 10,
    "MiningCurrency": null,
    "MinWorker": {
        "*": 10
    },
    "OutBuffer": null,
    "OutVariable": null,
    "PipelineVariable": null,
    "PoolBalancesUpdateInterval": 15,
    "PoolName": [
        "miningpoolhub",
        "zpool",
        "nicehash"
    ],
    "PowerPrices": {},
    "PricePenaltyFactor": 1,
    "ProfitabilityThreshold": 0,
    "Proxy": "",
    "Region": "US",
    "ReportStatusInterval": 90,
    "ShowAllMiners": false,
    "ShowAllPoolBalances": false,
    "ShowMinerWindow": false,
    "ShowPowerUsage": false,
    "SingleAlgoMining": false,
    "SSL": false,
    "SwitchingPrevention": 2,
    "UserName": "<-MPHUsername->",
    "Verbose": null,
    "Wallet": "<-MyBTCWallet->",
    "WarmupTime": 30,
    "WarningAction": null,
    "WarningVariable": null,
    "Watchdog": true,
    "WorkerName": "gpc",
    "Pools": {
        "MiningPoolHub": {
            "Region": "US",
            "API_Key": "<-MPHAPIKey->",
            "UserName": "At0micMac",
            "WorkerName": "gpc",
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "User": "At0micMac",
            "API_ID": 0
        },
        "NiceHash": {
            "Wallets": {
                "BTC": "3Ho18KNapR3H4WEojpnBiRGxcomAFuzmqw"
            },
            "IsInternalWallet": true,
            "Region": "US",
            "WorkerName": "gpc",
            "Worker": "gpc",
            "PricePenaltyFactor": 1
        },
        "MiningPoolHubCoins": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "User": "At0micMac",
            "API_ID": 0,
            "API_Key": "<-MPHAPIKey->"
        },
        "AHashPool": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "BlazePool": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "BlockMasters": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "HashRefinery": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "NLPool": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "PhiPhiPool": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "Ravenminer": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "YiiMP": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "ZergPool": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        },
        "Zpool": {
            "Worker": "gpc",
            "PricePenaltyFactor": 1,
            "Wallets": {
                "BTC": "<-MyBTCWallet->"
            }
        }
    },
    "Wallets": {
        "BTC": "<-MyBTCWallet->"
    },
    "VersionCompatibility": "3.3.0",
    "MinerStatusKey": "<-MyBTCWallet->",
    "Miners": {},
    "MinersLegacy": {
        "AMD_CPU_NVIDIA-FireiceCryptonight": {},
        "AMD_NVIDIA-BMiner": {},
        "AMD_NVIDIA-ClaymoreEthash": {},
        "AMD_NVIDIA-EthminerEthash": {},
        "AMD_NVIDIA-GminerEquihash": {},
        "AMD_NVIDIA-lolMinerEquihash": {},
        "AMD_NVIDIA-NanoMiner": {},
        "AMD_NVIDIA-NsgminerNeoscrypt": {},
        "AMD_NVIDIA-SgminerMTP": {},
        "NVIDIA-CcminerAlexis": {},
        "NVIDIA-CcminerDyn": {},
        "NVIDIA-CcminerKlausT": {},
        "NVIDIA-CcminerMTP": {},
        "NVIDIA-CcminerMTPNicehash": {},
        "NVIDIA-CcminerNevermore": {},
        "NVIDIA-CcminerPigeonCoinX16S": {},
        "NVIDIA-CcminerSp": {},
        "NVIDIA-CcminerSuprMiner": {},
        "NVIDIA-CcminerTpruvot": {},
        "NVIDIA-CcminerTrex": {},
        "NVIDIA-CcminerZenemy": {},
        "NVIDIA-CryptoDredge": {},
        "NVIDIA-DSTMEquihash": {},
        "NVIDIA-Ewbf2Equihash": {},
        "NVIDIA-EwbfEquihash": {},
        "NVIDIA-MiniZEquihash": {},
        "NVIDIA-NBMiner": {},
        "NVIDIA-ProgPoWminer": {},
        "NVIDIA-TTMiner": {},
        "NVIDIA-XmrigCryptonight": {},
        "NVIDIA-Zjazz": {}
    }
}
```

**Comment by UselessGuru** [comment]
@IronCraftMan 

I cannot see what is wrong with your config.
Please try 3.4.3:

https://github.com/MultiPoolMiner/MultiPoolMiner/issues/2350

**Comment by LinkReaction** [comment]
@UselessGuru Where can we get 3.4.3 now?

**Comment by UselessGuru** [comment]
> Where can we get 3.4.3 now?

Go for 3.4.4:
https://github.com/MultiPoolMiner/MultiPoolMiner/releases

**Comment by LinkReaction** [comment]
I'm getting the same config error though. 3.4.4 doesn't work for me.

**Comment by UselessGuru** [comment]
What if you remove (make a save copy) of the config file. Then start the default start.bat (as it is in the zip file)?

MPM will create a new default config file. After a minute or it should start mining to aaronsace's BTC account. If that is OK, then there is definitely something wrong with your start file.

**Comment by UselessGuru** [comment]
A workaournd for this problem is implemented in 3.4.5:

https://github.com/UselessGuru/MultiPoolMiner/releases/tag/3.4.5
